androidApp/src/androidMain/kotlin/com/br/kmmdemo/android/MainActivity.kt

```diff
@@ -1,17 +1,19 @@
 package com.br.kmmdemo.android
 
 import android.os.Bundle
+import com.br.kmmdemo.Platform
 import com.br.kmmdemo.main.MainView
-import com.br.kmmdemo.main.WeatherMapView
 import moe.tlaster.precompose.lifecycle.PreComposeActivity
 import moe.tlaster.precompose.lifecycle.setContent
+import org.koin.core.component.KoinComponent
+import org.koin.core.component.inject
 
-class MainActivity : PreComposeActivity() {
-
+class MainActivity : PreComposeActivity(), KoinComponent {
+    val platform: Platform by inject()
     override fun onCreate(savedInstanceState: Bundle?) {
         super.onCreate(savedInstanceState)
         setContent {
-            MainView()
+            MainView(platform.screenSize())
```

Comment:
Since this code is already platform specific (in the Android app module) do we need to abstract logic via platform file?